src/ecpool_worker.erl

```diff
@@ -24,7 +24,7 @@
 
 %% API Function Exports
 -export([ client/1
-        , exec/2
+        , exec/3
```

Comment:
To ensure hot upgrade safety, we should actually keep `exec/2`.
Otherwise if this module is upgraded to the new version,
calls from old version caller modules will crash
with `{EXIT, {undef, ....}}` (undefined function exception).

Reply:
Then let is crash :-)

I think we may need an API that accepts a timeout argument.

Reply:
Then let is crash :-)

I think we may need an API that accepts a timeout argument.

Reply:
we can keep exec/2 but annotate it with deprecated.

Reply:
The `exec/2` is also written by me some days ago, and so does the `exec_async/2`. No body in the older versions used these code, that's why I changed the APIs directly.